content/what-is/top-iac-tools.md

```diff
@@ -0,0 +1,129 @@
+type: what-is
+page_title: "Top Infrastructure as Code Tools"
+---
+
+Infrastructure as Code tools (IaC tools) let you automate the setup of your cloud resources without manually clicking around in your cloud web console. Instead of manually configuring resources, you write a script that specifies what you need, and the cloud provider sets it up for you. It's a great way to make infrastructure setup consistent and repeatable.
+
+But which tool should you use choose? We have our preferences and biases 😉 but let's go through the most popular and some less popular IaC tools to give you a lay of the land.
+
+We'll assess each tool based on the following criteria:
+
+| Criteria    | Description                                                                 |
+|-------------|-----------------------------------------------------------------------------|
+| Flexibility | Ability to abstract and create reusable components                          |
+| Multi-cloud  | Capability to manage resources across various cloud providers               |
+| State       | How the tool handles state tracking and management                          |
+| Integration | Compatibility with existing development workflows and CI/CD pipelines       |
+| Ecosystem   | Community strength, documentation quality, and support resources            |
+
+We'll cover modern multi-cloud tools, vendor and cloud-specific solutions, and some less common options.
+
+## Multi-Cloud Infrastructure as Code Tools
+
+First up, IaC tools that manage infrastructure across multiple cloud providers. You should pick one of these if you don't have any unusual requirements or legacy investments.
+
+### Pulumi
+
+This is the Pulumi website, so it's not an unbiased source of information. Still, Pulumi stands out as a versatile Infrastructure as Code (IaC) tool, offering the flexibility to manage infrastructure using a variety of programming languages.
+
+This flexibility allows developers to define infrastructure with real programming constructs, such as loops and functions, simplifying handling complex logic. Pulumi excels in multi-cloud support, integrating with major cloud providers such as AWS, Azure, Google Cloud, and many others.
+
+Pulumi's state management is easy to use, automatically handling infrastructure state and storing it in the Pulumi Service by default. This approach simplifies collaboration and reduces the risk of state corruption, with built-in state locking and encryption ensuring secure and consistent state management. For those who prefer more control, Pulumi also supports self-managed backends.
+
+Pulumi integrates with existing development workflows, allowing developers to use familiar tools and practices. Pulumi's ecosystem is growing rapidly, with many resources and integrations available. Comprehensive documentation and support, including tutorials and an active community forum, further bolster its appeal.
```

Comment:
```suggestion
Pulumi also integrates with existing development workflows, allowing developers to use familiar tools and practices. Pulumi's ecosystem is growing rapidly, with many resources and integrations available. Comprehensive documentation and support, including tutorials and an active community forum, further bolster its appeal.
```

"allowing developers to use familiar tools and practices" can you include some examples of familiar tools and practices? It also might be a good idea to list and link to some of the integrations.